sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/sas/BlobSasServiceVersion.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.sas;
+
+import com.azure.core.util.ServiceVersion;
+
+/**
+ * The versions of Azure Storage Blob Sas supported by this client library.
+ */
+public enum BlobSasServiceVersion implements ServiceVersion {
```

Comment:
Should we only ever support the latest version? if so we should probably remove this class